addons/fleet/data/mail_data.xml

```diff
@@ -1,10 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo><data noupdate="1">
+<odoo>
+    <data noupdate="1">
+        <record id="mail_act_fleet_contract_to_renew" model="mail.activity.type">
+            <field name="name">Contract to Renew</field>
+            <field name="icon">fa-car</field>
+            <field name="res_model_id" ref="fleet.model_fleet_vehicle_log_contract"/>
+        </record>
 
-    <record id="mail_act_fleet_contract_to_renew" model="mail.activity.type">
-        <field name="name">Contract to Renew</field>
-        <field name="icon">fa-car</field>
-        <field name="res_model_id" ref="fleet.model_fleet_vehicle_log_contract"/>
-    </record>
-
-</data></odoo>
+        <record id="mt_changed_driver" model="mail.message.subtype">
```

Comment:
mt_fleet_driver_updated (fleet prefix especially is important to know from which application it comes)